packages/analytics/analytics-geo-map/src/composables/useI18n.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="@vue/runtime-core" />
```

Comment:
issue: we shouldn't be using triple-slash type references

Reply:
Pushed a commit removing those but I think `prop typecheck` step fill fail in the CI